README.md

```diff
@@ -38,13 +38,13 @@ To help with translations and review head over to our translation [README.md](ht
 
 To run this project locally:
 
-1. First download the code from this repository
-2. Ensure you have NodeJS version 16.13.0 or higher installed
-3. Run `npm install` to download the required code libraries
-4. Run `npm run build` to build
-5. Run `npm run start` to spin up the development environment
-6. Open `http://localhost:3000` in your browser to access the site
-
+1. First, make sure you have NodeJS version 16.13.0 or higher installed. Check `node -v`. Node [installation instructions](https://nodejs.org/en/download/package-manager/)
+3. Download this repository `git clone https://github.com/saving-satoshi/saving-satoshi.git`
```

Comment:
I think you skipped number `2`

Reply:
Hmmmm, when I click the readme doc and look at the other view the '2.' Is there.